clients/cloud/java/README.md

```diff
@@ -1,95 +1,118 @@
-# Java Producer and Consumer for Confluent Cloud
+# Overview
 
-This directory includes projects demonstrating how to use the Java producer and consumer with Confluent Cloud.
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the Java Producer and Consumer, and Kafka Streams API.
 
-For more information, please see the [application development documentation](https://docs.confluent.io/current/api-javadoc.html)
 
+# Prerequisites
 
-# Quickstart
+* Java 1.8 or higher to run the demo application
+* Maven to compile the demo application
+* [Confluent Cloud CLI](https://docs.confluent.io/current/cloud/cli/install.html) installed on your machine. It is provided as part of the Confluent Platform package or may be [installed separately](https://docs.confluent.io/current/cloud/cli/install.html).
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* [Initialize](https://docs.confluent.io/current/cloud/cli/multi-cli.html#connect-ccloud-cli-to-a-cluster) your local Confluent Cloud configuration file using the `ccloud init` command, which creates the file at `$HOME/.ccloud/config`.
 
-Before running the examples, **you should setup the Confluent Cloud CLI** by running the [quickstart](https://docs.confluent.io/current/quickstart/cloud-quickstart.html#cloud-quickstart).
+# Example 1: Hello World!
 
+In this example, the producer writes Kafka data to a topic in Confluent Cloud. 
+Each record has a key representing a username (e.g. `alice`) and a value of a count, formatted as json (e.g. `{"count": 0}`).
+The consumer reads the same topic from Confluent Cloud and keeps a rolling sum of the counts as it processes each record.
+The Kafka Streams API reads the same topic from Confluent Cloud and does a stateful sum aggregation, also a rolling sum of the counts as it processes each record.
 
-1. Create a topic called `page_visits`:
+1. Run the producer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the topic name:
 
 	```shell
-	# Create page_visits topic
-	$ ccloud topic create page_visits
-	```
-	You should see
+	# Build the client examples
+	$ mvn clean package
 	
+	# Run the producer
+	$ mvn exec:java -Dexec.mainClass="io.confluent.examples.clients.ProducerExample" \
```

Comment:
should there be a "cloud" somewhere in that namespace?

Reply:
Ack to this and all references